### ISSUE TITLE
Snapshot name issue

### ISSUE THREAD

**Issue by ussuritiger** [suchipi/chai-jest-snapshot#2]
Hi Stephen, first of all thank you for such a nice Chai plugin. Now I can easily extend my testing setup with snapshot testing of React components without switching to Jest (nothing bad about Jest but I have other tools in place already).

While writing tests with chaiJestSnapshot assertion I've encountered weird issue: I have four tests inside one `describe` block and in every test I do snapshot assertion. Snapshot file name is the same for all four tests and is set for entire `describe` block. Snapshot name is set per test four time and unique. When I run test without or with already rendered snapshot file I have only two snapshots out of four inside the snapshot file. When I try to change the name of snapshots and rerender snapshots (or render a new snapshot file), some of snapshots appear but I could not see a logic there.

Do you have any idea why this can happen?

**Comment by suchipi** [comment]
I don't know why it'd happen off the top of my head. Could you provide a code example I could run some tests against?

**Comment by ussuritiger** [comment]
This is component:

```javascript
import React, { Component, PropTypes } from 'react';

class TestComponent extends Component {
    render() {
        const {number} = this.props;

        let string = '';

        switch (number) {
            case 1:
                string = 'one';
                break;
            case 2:
                string = 'two';
                break;
            case 3:
                string = 'three';
                break;
            case 4:
                string = 'four';
                break;
        }

        return (
            <div>{string}</div>
        );
    }
}

TestComponent.propTypes = {
    number: PropTypes.number.isRequired
};

export default TestComponent;
```

This is test:
```javascript
import path from 'path';
import chai, {expect} from 'chai';
import chaiJestSnapshot from 'chai-jest-snapshot';
import React from 'react';
import {shallow} from 'enzyme';
import TestComponent from 'TestComponent';

describe('TestComponent.js', () => {
    chai.use(chaiJestSnapshot);

    const snapshotFileName = path.join(__dirname, 'TestComponent.spec.js.snap');

    describe('#render', () => {
        it('renders one', () => {
            let snapshotName = 'one';

            let output = shallow(
                <TestComponent number={1}/>
            );

            expect(output.html()).to.matchSnapshot(snapshotFileName, snapshotName);
        });

        it('renders two', () => {
            let snapshotName = 'two';

            let output = shallow(
                <TestComponent number={2}/>
            );

            expect(output.html()).to.matchSnapshot(snapshotFileName, snapshotName);
        });

        it('renders three', () => {
            let snapshotName = 'three';

            let output = shallow(
                <TestComponent number={3}/>
            );

            expect(output.html()).to.matchSnapshot(snapshotFileName, snapshotName);
        });

        it('renders four', () => {
            let snapshotName = 'four';

            let output = shallow(
                <TestComponent number={4}/>
            );

            expect(output.html()).to.matchSnapshot(snapshotFileName, snapshotName);
        });
    });
});
```

And this is content of snapshot file:

```javascript
exports[`four`] = `"<div>four</div>"`;

exports[`one`] = `"<div>one</div>"`;
```

**Comment by suchipi** [comment]
Thanks for the report. This should be fixed in `0.1.3`. Please give it a try and let me know how it works for you.

Thanks,
Stephen

**Comment by ussuritiger** [comment]
Works like a charm. Thank you for quick reaction.
